package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "svelte",
-  "version": "3.29.7",
+  "version": "3.31.0",
```

Comment:
I think you didn't rebase properly. This line shouldn't be changed

Reply:
I did a merge from parent-2 instead of rebase, local diff shows everything's fine, but Github diff acts just weird, so in the latest commit, I just squashed everything on top of latest master.